src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  showModal: (arg: boolean) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, showModal }) => {
+  const { id: todoId, title: todoTitle, completed, userId } = todo;
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(userId).then(setUser);
+  }, []);
```

Comment:
Consider adding error handling for the `getUser` promise. This will help manage any potential errors that occur during the API call and improve the robustness of your application.